### ISSUE TITLE
Sources of stochasticity

### ISSUE THREAD

**Issue by sdhrkelam** [OpenRobotLab/AnySplat#81]
Thank you for sharing this codebase. I am currently experimenting with it, but I am facing reproducibility issues. When I run the same experiment twice — using the same scene, the same set of context views, and with data augmentation and voxelization explicitly disabled — the loss curves still do not match between runs.

I would like to confirm whether there are other sources of randomness in the training pipeline that should be controlled to ensure reproducible results.
I am using these flags in my command
    dataset.dl3dv.augment=false \
    dataset.dl3dv.intr_augment=false \
    dataset.dl3dv.input_image_shape=[448,448] \
    model.encoder.voxelize=false

**Comment by daniyar-niantic** [comment]
Hi. Not an author, but maybe the randomness is due to dataloader workers not returning the same batches between runs.
